src/features/click-to-load.js

```diff
@@ -632,44 +596,25 @@ function getLearnMoreLink (mode) {
 }
 
 /**
- * Reads and stores a set of styles from the original tracking element, and then returns it.
- * @param {Element} originalElement Original tracking element (ie iframe)
- * @param {DuckWidget} widget The widget Object.
- * @returns {{[key: string]: string[]}} Object with styles read from original element.
+ * Resizes the target element to match the current size of the source element.
+ * @param {Element} sourceElement
+ * @param {Element} targetElement
  */
-function getOriginalElementStyle (originalElement, widget) {
-    if (widget.originalElementStyle) {
-        return widget.originalElementStyle
-    }
-
-    const stylesToCopy = ['display', 'visibility', 'position', 'top', 'bottom', 'left', 'right',
-        'transform', 'margin']
-    widget.originalElementStyle = {}
-    const allOriginalElementStyles = getComputedStyle(originalElement)
-    for (const key of stylesToCopy) {
-        widget.originalElementStyle[key] = {
-            value: allOriginalElementStyles[key],
-            priority: originalElement.style.getPropertyPriority(key)
-        }
+function resizeElementToMatch (sourceElement, targetElement) {
+    let { height, width } = sourceElement.getBoundingClientRect()
+
+    // The declarativeNetRequest API "collapses" (hides) blocked elements. When
+    // that happens, fall back to getComputedStyle to figure out the element's
+    // dimensions.
+    if (height === 0 && width === 0) {
+        ({ height, width } = window.getComputedStyle(sourceElement))
+    } else {
+        height = height + 'px'
+        width = width + 'px'
     }
 
-    // Copy current size of the element
-    const { height: heightViewValue, width: widthViewValue } = originalElement.getBoundingClientRect()
-    widget.originalElementStyle.height = { value: `${heightViewValue}px`, priority: '' }
-    widget.originalElementStyle.width = { value: `${widthViewValue}px`, priority: '' }
-
-    return widget.originalElementStyle
-}
-
-/**
- * Copy list of styles to provided element
- * @param {{[key: string]: string[]}} originalStyles Object with styles read from original element.
- * @param {Element} element Node element to have the styles copied to
- */
-function copyStylesTo (originalStyles, element) {
-    const { display, visibility, ...filteredStyles } = originalStyles
-    const cssText = Object.keys(filteredStyles).reduce((cssAcc, key) => (cssAcc + `${key}: ${filteredStyles[key].value};`), '')
-    element.style.cssText += cssText
```

Comment:
I don't think we can remove the original function completely. I remember I added this to try and copy relevant CSS styles from the original element so that our placeholders sit better on the page. In particular, videos that have `position: absolute` will appear on top of empty space that the page had originally intended for the video. Since we by default use `position: relative`, our placeholder will then stack bellow this empty area. And we can't just always use `position: absolute` because not all pages are using styles like this, that's why we need to try and copy relevant styles.
Here is an example that appeared during Design Review: https://www.raspberrypi.com/news/this-robots-extra-arm-gives-it-whole-body-control/
This is how it looks now:
![image](https://user-images.githubusercontent.com/14364481/228266897-5bf9a0f9-a347-4d8c-99b4-82743268a1cf.png)


This is how it looked previously:
![image](https://user-images.githubusercontent.com/14364481/228266518-b5843bf2-15fe-4f3b-9ab1-6edc16c7f2b3.png)

Reply:
Great catch, thanks for pointing this out. I think I've sorted this now, that example renders OK for me on Chrome MV2 and MV3.

Reply:
Great fix 🙌